client/app/styles/app.scss

```diff
@@ -187,6 +189,14 @@ $sticky-sidebar-offset: 6rem + rem-calc(20);
       margin-bottom: 5px;
     }
   }
+
+  h3 {
```

Comment:
Coincidentally I just used an h3 under the equitable-development-reporting form, but the h3 I created needs to stay black. Is there a way you can make this more specific? Maybe create an ID for just the auto response text?

Reply:
sure, I added a class to help differentiate things.

Reply:
TY! sounds good